portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SidebarMenu.testcase

```diff
@@ -30,12 +30,21 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8158 | Test Summary: Switch the property which was selected by default"
-	@ignore = "true"
 	@priority = "5"
 	test CanSwitchPropertyWhichWasSelectedByDefault {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
 
-		// TODO LRAC-8158 CanSwitchPropertyWhichWasSelectedByDefault pending implementation
+		ACUtils.launchAC();
+
+		AssertTextNotEquals(
+			locator1 = "ACSidebar#CURRENT_PROPERTY",
+			value1 = "Liferay DXP");
```

Comment:
@91nilo, I think you'd better use the propertyName variable instead of Liferay DXP, because in CI this name will be different.